ruby_connectors/lib/ruby_connectors/temporal/activities/read_api_data_activity.rb

```
@@ -0,0 +1,48 @@
+              page_number: params["page"].to_i }
+          rescue StandardError => e
+            ::Temporal.logger.error("ReadDataActivity failed: #{e.message}")
+
+            {
+              workflow_id: params["workflow_id"],
+              current_page: params["page"].to_i,
+              error: e.message,
+              status: "error"
+            }
+          end
+        end
```

rubocop (1.68.0)

[convention] Assignment Branch Condition size for execute is too high. [<7, 24, 2> 25.08/17]

(Metrics/AbcSize)